src/platforms/react-native/advanced-setup.mdx

```diff
@@ -140,7 +140,7 @@ public class MainApplication extends Application implements ReactApplication {
 
 Add the following to your `settings.gradle` file:
 
-```java
+```groovy
```

Comment:
```suggestion
```groovy {filename:android/app/build.gradle}
```

Reply:
Looks like this should be `settings.gradle` and not `app/build.gradle`
```suggestion
```groovy {filename:android/settings.gradle}
```